docs/scripts/formattedTSDemos.js

```diff
@@ -1,55 +1,113 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+/* eslint-disable no-continue */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
```

Comment:
These are here for a reason. Could you  adjust the code to work without those (except no-console, we obviously need that still) or at least add a rationale for disabling these?

Reply:
@eps1lon Reworked the implementation, got rid of all of them (except no-console of course)